extensions/cornerstone-dicom-seg/src/viewports/OHIFCornerstoneSEGViewport.tsx

```diff
@@ -160,6 +168,8 @@ function OHIFCornerstoneSEGViewport(props) {
       viewportIndex,
       segDisplaySet,
     }).then(isHydrated => {
+      storePresentationState();
```

Comment:
Why do we store the presentation after/once the SEG is hydrated? I would think that would be too late with respect to fixing issue #3410, but maybe this is for something else. This comment applies to other similar changes in this PR.

Reply:
I added comment, sorry it was not clear you are right

Reply:
Sorry, but I am still a bit confused. The `then` callback is called AFTER the SEG is hydrated, correct? If so, then the first line of the comment you added seems a bit contradictory. But maybe I am missing something.

Reply:
Thanks for your changes. However, considering the issue seemed fixed even before the most recent changes, I am not sure if the changes actually contribute to the fix. Just wanted to mention this.